src/Symfony/Component/Messenger/HandleTrait.php

```diff
@@ -31,16 +32,17 @@ trait HandleTrait
      * the last one usually returning the handler result.
      *
      * @param object|Envelope $message The message or the message pre-wrapped in an envelope
+     * @param StampInterface[] $stamps Stamps to be set on the Envelope which are used to control middlewares behaviours
```

Comment:
```suggestion
     * @param StampInterface[] $stamps Stamps to be set on the Envelope which are used to control middlewares behaviors
```

Reply:
Looks like behavior was more used then behaviour in the code base did update it. Thx for the review!